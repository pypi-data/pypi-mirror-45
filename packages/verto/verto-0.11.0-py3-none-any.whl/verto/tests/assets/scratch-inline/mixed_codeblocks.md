How about some scratch: `scratch:say [Hello]` which is the same as the python expression: `print('Hello')`
