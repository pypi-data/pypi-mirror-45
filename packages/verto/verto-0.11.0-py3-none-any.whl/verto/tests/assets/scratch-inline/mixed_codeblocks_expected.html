<p>How about some scratch: <object data="{% static 'scratch-blocks-9dabe0bac28bc3a143cfb19c2e5d7f46aae62b3d793166a56665a789d0df5bf7.svg' %}" type="image/svg+xml"></object> which is the same as the python expression: <code>print('Hello')</code></p>
